Resources/doc/guide.rst

```diff
@@ -0,0 +1,211 @@
+Beginners guide
+===============
+
+Befor starting this, you should have already installed:
+
+Text editor (Phpstorm, Vim, SublimeText, BlueFish..),
+
+Symfony,
+
+Composer,
+
+Git,
+
+ElasticSearch (and launched it, so you can go to localhost:9200 and see it's working)
+
+Create new Symfony project
+--------------------------
+Open Terminal, go to your working directory or create new
+
+.. code:: bash
+
+    mkdir Sites
+    cd Sites
+
+Create new Symfony project
+
+.. code:: bash
+
+    symfony new project
+
+Setup ElasticSearchBundle
+-------------------------
+
+Download ElasticsearchBundle. This version ``"~0.1"`` can be changed to newer.
+
+.. code:: bash
+
+    cd project
+    composer require ongr/elasticsearch-bundle "~0.1"
+
+Open Text editor and navigate to ``Sites/project/app/AppKernel.php``. Enable ElasticSearchBundle by adding this one line:
+
+.. code:: php
+
+    <?php
+    // app/AppKernel.php
+
+    public function registerBundles()
+    {
+        $bundles = array(
+            // ...
+            new ONGR\ElasticsearchBundle\ONGRElasticsearchBundle(),
+        );
+    }
+
+Go to ``app/config/config.yml``, add configuration at the very bottom:
+
+.. code:: php
+
+    ongr_elasticsearch:
+        connections:
+            default:
+                hosts:
+                    - { host: 127.0.0.1:9200 }
+                index_name: product
+                settings:
+                    refresh_interval: -1
+                    number_of_replicas: 1
+        managers:
+            default:
+                connection: default
+                mappings:
+                    - AcmeDemoBundle
+
+Creating Database for ElasticSearch
+-----------------------------------
+
+Go to ``src/Acme/DemoBundle`` and create new Folder ``Document``. It must be named as "Document". ONGR points to exactly this directory.
```

Comment:
I plan to include this later, how to change Document's folder.  I could edit this line like "ONGR points to exactly this directory as default".